meta/client/tidbclient/bucket.go

```diff
@@ -263,20 +263,22 @@ func (t *TidbClient) DeleteBucket(bucket Bucket) error {
 	return nil
 }
 
-func (t *TidbClient) UpdateUsage(bucketName string, size int64, tx interface{}) (err error){
+func (t *TidbClient) UpdateUsage(bucketName string, size int64, tx interface{}) (err error) {
 	var sqlTx *sql.Tx
 	if tx == nil {
 		tx, err = t.Client.Begin()
+
 		defer func() {
+			if err == nil {
+				err = sqlTx.Commit()
+			}
 			if err != nil {
 				sqlTx.Rollback()
-			} else {
-				sqlTx.Commit()
 			}
 		}()
-	} else {
-		sqlTx, _ = tx.(*sql.Tx)
 	}
+	sqlTx, _ = tx.(*sql.Tx)
```

Comment:
Can we use type to constrain `tx`, so we do not need to do this type-assertion? 
From `tx interface{}` to `tx *sql.Tx`

Reply:
If only support tidb, chang tx type from interface{} to *sql.Tx seems ok

Reply:
"tx type" will be modified in another PR